public/app/features/integrated-alerting/components/NotificationChannel/AddNotificationChannelModal/AddNotificationChannelModal.messages.ts

```diff
@@ -0,0 +1,17 @@
+export const Messages = {
+  title: 'Add Notification Channel',
+  fields: {
+    name: 'Name',
+    type: 'Type',
+    addresses: 'Addresses',
+    addressesPlaceholder: 'example1@percona.com\nexample2@percona.com\nexample3@percona.com',
+    routingKey: 'Routing key',
+    serviceKey: 'Service key',
+    channel: 'Channel',
+  },
+  addAction: 'Add',
+  cancelAction: 'Cancel',
+  emailOption: 'Email',
+  pagerDutyOption: 'Pager Duty',
+  slackOption: 'Slack',
+};
```

Comment:
Weren't we using the `message` file on the outer folder (`integrated-alerting`)?

Reply:
I got the idea we agreed to one per component by out discussion.

Reply:
I had the opposite feeling xD Let's ask @atymchuk, in the meantime I'm going to approve this, since this is a nitpick

Reply:
I'd put the message file close to the component. For two reasons:
1/ less inter-component dependencies (easier to refactor)
2/ clean  imports

Reply:
Good, I like this!